app/javascript/crayons/Pills/__stories__/Pills.stories.jsx

```diff
@@ -0,0 +1,121 @@
+import { h } from 'preact';
+import { Pill } from '..';
+import PillsDoc from './Pills.mdx';
+import CakeIcon from '@images/cake.svg';
+import XIcon from '@images/x.svg';
+import HideIcon from '@images/eye-off.svg';
+
+export default {
+  component: Pill,
+  title: 'Components/Pills',
+  parameters: {
+    docs: {
+      page: PillsDoc,
+    },
+  },
+  argTypes: {
+    element: {
+      control: {
+        type: 'select',
+        options: {
+          button: undefined,
+          a: 'a',
+          span: 'span',
+          li: 'li',
+        },
+      },
+      description:
+        'Pills can be used in a bunch of different contexts, therefore it is important to use appropriate element to represent it in the markup.',
+      table: {
+        defaultValue: { summary: 'button' },
+      },
+    },
+    iconLeft: {
+      control: false,
+      description:
+        'If possible, this icon should represent the actual data associated with the Pill.',
+      table: {
+        defaultValue: { summary: 'LockIcon' },
+      },
+    },
+    iconRight: {
+      control: false,
+      description:
+        'If possible, this icon should represent the action associated with the Pill. If there is no action associated, consider not using this icon.',
```

Comment:
It's just a random thought but we could name the icons something like `actionIcon` vs `descriptionIcon` instead of left/right to try and guide people to use the right one 🤔  (just a thought - feel free to ignore)

Reply:
Oh, I like this idea a lot!

Reply:
Done.